apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Operations.kt

```diff
@@ -246,7 +265,7 @@ fun <D : Operation.Data> Operation<D>.composeJsonRequest(
     sink: BufferedSink,
     customScalarAdapters: CustomScalarAdapters,
 ) {
-  val composer = ApolloHttpRequestComposer("unused")
+  val composer = DefaultHttpRequestComposer("unused")
```

Comment:
This is purely cosmetic because we already have a lot of `DefaultXyz` for classes that implement an interface. 

`ApolloXyz` on the other hand are for high level APIs, which I feel this API doesn't belong.

I'm happy to revert if there is any objection